### PR TITLE
docs(fix): updated documentation to fix inconsistencies

### LINKED ISSUE #198: Update README.md to fix inconsistencies
Inconsistencies found so far:

- [ ] inconsistencies in Database variables information for .env
- [ ] example.env needs to match [README.md](http://readme.md/) instructions
- [ ] make sure .env information is linked in Docker instructions.
- [ ] instructions on installing docker (using these links for instructions for [Windows](https://docs.docker.com/desktop/install/windows-install/) and [Mac](https://docs.docker.com/desktop/install/mac-install/)

### PR DESCRIPTION
**Pull request by jimbandrews**
### **1. Targeted Issue** ###

*(The first step to a good solution is knowing the problem. Please link the issue here.)*

Issue #... or
Ticket #...

https://trello.com/c/aBYGPWSM/152-update-readmemd-to-fix-inconsistencies

### **2. Overview of Solution** ###

*(You've put in the hard work, now's your chance to tell us about it. What's your solution in a nutshell?)*

Added Docker install instructions.
Updated Environment Variables to match example.env.
Added $ to beginning of bash commands.

### **3. Tools Used** ###

*(Share the tools, libraries, or techniques you've used in your solution.)*

### **4. Testing Strategy** ###

*(No code is complete without tests. Outline what you've tested and how.)*

### **5. Future Implications** ###

*(Let's anticipate potential effects of your changes. Are there any potential issues down the line?)*

### **6. Screenshots** ###

*(A picture is worth a thousand words. Include screenshots of your changes if relevant.)*

### **7. Code Reviewers** ###

*(Who else is in the loop? Mention your team or other contributors here.)*

---

Before you go...

- [x ]  I have linked the relevant issue
- [x ]  I have provided an overview of my solution
- [ ]  I have shared the tools I've used
- [ ]  I have outlined my testing strategy
- [ ]  I have considered future implications of my changes
- [ ]  I have included relevant screenshots
- [ ]  I have kept my team in the loop

---

### CODE REVIEW

File: example.env [hunk 1]
```diff
@@ -1,18 +1,24 @@
...
+
 EMAIL_HOST=Your email host
 EMAIL_HOST_USER=Example
 EMAIL_HOST_PASSWORD=Example
+
 SENTRY_DSN=Example
-CELERY_BROKER_URL=Example
-CELERY_RESULT_BACKEND=Example
 
 (If you do not plan to create an Email server, AWS S3 bucket,
```
Comment (esparr) [inline example.env:23]: I wonder what this is referencing? Perhaps settings.py?

**Review by esparr — CHANGES_REQUESTED** [review]
This looks really good. It is nice to see our docs get fleshed out so nicely. I added a couple inline comments but also while reading through the README.md I found a couple misspellings we have all missed and then have a couple of suggestions for other improvements. I am okay not adding in more details about the database setup without docker since I am hoping most folks will lean on docker now. 

Line 27 - misspelling of availability
Line 28 - misspelling of mentors, availability
After Line 32 - Add subheader ‘## Local setup without containers` or something similar
Around Line 63 - Instructions on setting up the postgres database locally would be helpful.
Line 78 - convert header to ### size
Line 266 - 💡***Note*** before information

Thank you for all you have put into making our documentation shine.

**Comment by jimbandrews** [comment]
Thanks for the feedback, @esparr. Comments have been addressed!

**Review by GitLukeW — APPROVED** [review]
Thanks for the changes! @esparr have all your change request be resolved?

**Review by esparr — APPROVED** [review]
Looks good! Thanks for tackling our documentation. It is looking amazing.

**Pull request merged**